### PR TITLE
fix(components/Virtualized): custom renderer to hide the header

### PR DESCRIPTION
**Pull request by jare-talend**
**What is the problem this PR is trying to solve?**
Actually, the current list can hide the header if indicate, not Virtualized
**What is the chosen solution to this problem?**
Create a HOC to custom the header renderer

Also, i remove the story ellipsis. The class isn't used and add the story on scroll

**Please check if the PR fulfills these requirements**
- [ ] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/src/VirtualizedList/HeaderRenderer/HeaderRenderer.component.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+			className={classnames}
+			key="label"
+			title={label}
+		>
+			{label}
+		</span>);
+	}
+
+	HeaderRenderer.propTypes = {
+		...WrappedComponent.propTypes,
```
Comment (jmfrancois) [inline packages/components/src/VirtualizedList/HeaderRenderer/HeaderRenderer.component.js]: the werapped component isn't use here, so it should be label

File: packages/components/src/VirtualizedList/HeaderRenderer/HeaderRenderer.component.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+/**
+ * Higher order header-renderer that wrap the provided header renderer.
+ */
+function getHeaderRenderer(WrappedComponent, hideHeader) {
+	function HeaderRenderer(props) {
+		const {
+			label,
+		} = props;
+
+		const classnames = classNames('ReactVirtualized__Table__headerTruncatedText', 'sr-only');
```
Comment (jmfrancois) [inline packages/components/src/VirtualizedList/HeaderRenderer/HeaderRenderer.component.js]: what the className ReactVirtualized__Table__headerTruncatedText stand for ?

File: packages/components/src/VirtualizedList/HeaderRenderer/HeaderRenderer.component.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+/**
+ * Higher order header-renderer that wrap the provided header renderer.
+ */
+function getHeaderRenderer(WrappedComponent, hideHeader) {
+	function HeaderRenderer(props) {
```
Comment (jmfrancois) [inline packages/components/src/VirtualizedList/HeaderRenderer/HeaderRenderer.component.js]: I'm sure this component can be declare outside of getHeaderRenderer.

**Comment by jsomsanith** [comment]
The feature in current list (https://github.com/Talend/ui/blob/ce1d9098b15aba04fa0b0f41ddc8789ae7bea6df/packages/components/src/List/DisplayTable/DisplayTable.component.js#L160) was to disable sort on columns like actions columns.
This feature is covered by `disableSort` attribute in `<VirtualizedList.Content>`

**Pull request closed without merging**